charts/akeyless-gateway/values.yaml

```diff
@@ -113,6 +113,42 @@ globalConfig:
         cpu: 250m
         memory: 256Mi
 
+cacheHA:
+  enabled: false
+## TODO: Need to find the right value to control the Redis port
+
+  nameOverride: "cache-ha"
+
+  ## In case cacheHA is enabled in the Gateway, and the encryptionKeyExistingSecret parameter has a value
+  ## Akeyless will use this specified encryption key and store it securely within Akeyless Gateway.
+  ## If the encryptionKeyExistingSecret parameter is empty or not specified,
+  ## Akeyless will automatically generate a new encryption key and a new ServiceAccount for K8s.
+  ## for more information: https://docs.akeyless.io/docs/advanced-chart-configuration#cache-configuration
+  ##
+  encryptionKeyExistingSecret:
+  tls:
+    enabled: true
+    authClients: false
+    autoGenerated: true
+
+  sentinel:
+    enabled: true
+
+  ## Required for cacheHA, a storage class with the ReadWriteOnce access mode
+  global:
+    defaultStorageClass: ""
+
+  master:
+    pdb:
+      create: false
+      minAvailable: ""
+      maxUnavailable: ""
+  replica:
+    pdb:
+      create: false
+      maxUnavailable: ""
+      minAvailable: ""
+
```

Comment:
my pdb will enter in other drop, check with @Eyal-po. if not entering pdb, you need to remove all pdb code